### ISSUE TITLE
Internal Errors When Launching Apps

### ISSUE THREAD

**Issue by GHuser0724** [sphinx02/FireStarter#4]
Hello.  I just installed Firestarter on my Amazon FireTV box and love it.  However, I'm having a couple problems with it.  When I try to launch some of the apps I've installed, especially ones I installed through the Amazon Appstore,  I get an error message that says, "Amazon Appstore: internal failure.  An internal error occurred, please try launching the app again".  This happens with the Youtube app, the Smithsonian app and others.  Kodi and Netflix work OK.

The other problem is I cannot launch ANY apps from the Amazon homescreen.  When I press the select button to launch whatever app I'm trying to launch, nothing happens, as if I never pressed the button at all.  

I rebooted the FireTV to see if that would fix the problems but it has not.  Thanks for any help you can give.

**Comment by GHuser0724** [comment]
I uninstalled FireStarter.  Now I cannot even launch apps from the Amazon launcher. Help!  Your app has seriously messed up my FireTV!

**Comment by sphinx02** [comment]
Hi, I am sorry you are facing these issues in your FireTV.. I hope we get it back working! From what you described in your first post I am quite sure the cause of your problems is not FireStarter. FireStarter does e.g. not affect the way amazon is launching Apps in any way.. I recommend that you first reset your FireTV to factory defaults (in FireTV settings) and Start then with a clean setup.. Then check after each App Installation if it is still working.. Let me know if I can further help you!

Best regards
Lukas

**Comment by GHuser0724** [comment]
Hi Lukas,

Thx for your response.  Yeah, I don't know what happened with my FireTV.  I just know I started getting these errors after installing Firestarter.  So I did reset the box back to its factory defaults and that fixed the issues I was having.  May try Firestarter again, though the new version of Kodi (15.0) lets you access your apps in much the same way as Firestarter does without having to go to the Amazon home screen.

Thanks for the quick response.  Will let you know if I encounter any other issues.

Kind regards,
Dan
